api/src/main/java/datawave/microservice/querymetric/BaseQueryMetric.java

```diff
@@ -944,6 +944,8 @@ public void setError(Throwable t) {
             this.setErrorCode(qe.getErrorCode());
```

Comment:
It is possible for a QueryException be created with a blank error code. I don't believe there any actual cases where we create a QueryException with a blank error code, but for future proofing, please set the error code here to 500-1 if a non-blank one is not present within the exception itself.

Reply:
Added in latest commit